meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend

```diff
@@ -2,3 +2,28 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 EXTRA_OENPM:witherspoon-tacoma = "-- --mode ibm"
 EXTRA_OENPM:p10bmc = "-- --mode ibm"
+
+inherit obmc-phosphor-systemd
+
+SRC_URI:append:p10bmc = " \
+	file://changeLogo.sh \
+	file://change-logo.service \
+	file://inspur-login-logo.svg \
+	file://inspur-logo-header.svg \
+	"
+
+SYSTEMD_SERVICE:${PN}:p10bmc = " change-logo.service"
+
+do_install:append:p10bmc() {
+    if [ -e ${WORKDIR}/inspur-login-logo.svg ]; then
```

Comment:
Why is the "if" check needed? File should always be there.

Reply:
Sure, Removed